FuelLabs/fuel-indexer
# chore: general plugin cleanup

### Description
- Updates new beta net references to `beta-3` (not node-beta-3.fuel.network)
- Prints the PID of `forc index start` so a user can kill the proc without having to find it (it runs in the background so Ctrl+C won't work)
- Replaces all 127.0.0.1 references with localhost
- Fixes the examples
  - docker-compose in block explorer was updated with the new `run` command
  - Docs were updated with new example names
- README was updated

### Testing steps
- [ ] Test out the examples
  - With and without docker
- [ ] If manually testing commands
  - Ensure `forc index start` is being built with your local `fuel-indexer` at /path/to/fuel-indexer/target/release/fuel-indexer, _not_ the indexer in your `PATH`

### Changelog
- chore: general plugin cleanup and example updates
